program.py

```diff
@@ -181,11 +181,11 @@ def extract_examples(self) -> list[('DafnyProgram', str)]:
             line_number, annotation, program = program.strip_first_annotation()
             if line_number is None:
                 break
-            examples.append((program, annotation))
+            example_program = program if not localized else program.insert(line_number, "/*[CODE HERE]*/")
```

Comment:
Maybe reference CODE_HERE_MARKER from `completion.py` just to make sure these remain the same string? It looks right for now, but just because this is easy to mess up later in one place and forget the others.